### PR TITLE
feat(context-pad): render tooltips

### LINKED ISSUE #18: Use properties panel tooltips for context pad
### What should we do?

Use properties panel tooltips in context pad:
Screenshot 2023-12-22 at 10 17 23

Requirements for tooltip integration:
- [x] Export `Tooltip` component in `@bpmn-io/properties-panel` (https://github.com/bpmn-io/properties-panel/pull/321)
- [ ] Support configuration of tooltip orientation (https://github.com/camunda/improved-canvas/pull/20)

### Why should we do it?
As part of #5

### PR DESCRIPTION
**Pull request by smbea**
Needs https://github.com/bpmn-io/properties-panel/pull/321 ✅
Closes #18

### CODE REVIEW

File: lib/features/contextPad/ImprovedContextPad.js [hunk 1]
```diff
@@ -31,4 +47,35 @@ ImprovedContextPad.prototype._getPosition = function(target) {
...
+  const entries = domQueryAll('.entry', current.pad.html);
+
+  entries.forEach((entry)=> {
+
+    const position = getTooltipPosition();
+
+    // remove native tooltip
+    const title = entry.getAttribute('title');
+    entry.setAttribute('data-title', title);
+    entry.removeAttribute('title');
```
Comment (marstamm) [inline lib/features/contextPad/ImprovedContextPad.js:61]: I see we are removing the title and only add it as a private `data-` field. I wonder if this has implications for accessibility? What do you think about this?
```suggestion
    entry.setAttribute('aria-label', title);
    entry.removeAttribute('title');
```

It is easy for us and might help someone :)

File: lib/features/contextPad/ImprovedContextPad.js [hunk 2]
```diff
@@ -6,17 +6,33 @@ import {
...
+
+export default function ImprovedContextPad(contextPad, eventBus, canvas) {
 
   insertCSS('base', baseCSS);
   insertCSS('context-pad', contextPadCSS);
 
   contextPad._getPosition = this._getPosition;
+
+  eventBus.on('contextPad.open', this._renderTooltips);
+
```
Comment (marstamm) [inline lib/features/contextPad/ImprovedContextPad.js:35]:
```suggestion
```

**Review by marstamm — COMMENTED** [review]
Works great! One small comment on the title attribute below.

File: lib/features/contextPad/ImprovedContextPad.js (hunk 1 above)
Comment (smbea) [reply lib/features/contextPad/ImprovedContextPad.js:60]: I think that's fair enough. Updated

**Review by marstamm — APPROVED** [review]

**Pull request merged**